### PR TITLE
✨ Build Profile page

### PR DESCRIPTION
**Pull request by fabiglahn**
## Description

I built the profile page. 
(Note: If there was time, there could be another database with the classes (which there's not)) 

## Design
Bildschirmfoto 2021-12-09 um 12 05 08

### CODE REVIEW

File: src/Pages/Profile/Profile.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import ClassCard from "../../Components/ClassCard/ClassCard";
+import NavigationBar from "../../Components/NavigationBar/NavigationBar";
+import ProfilePicture from "../../Components/ProfilePicture/ProfilePicture";
+import styles from "./Profile.module.css";
+
+function Profile(): JSX.Element {
+  return (
+    <div className={styles.container}>
```
Comment (derjubu) [inline src/Pages/Profile/Profile.tsx]: `<article>` might be a bit more semantic here

**Review by derjubu — APPROVED** [review]
LGTM 🚀

File: src/Components/ProfilePicture/ProfilePicture.module.css [hunk 2]
```diff
@@ -1,28 +1,28 @@
...
-.stripe {
+  margin-top: 5rem;
   background-color: var(--color-primary-accent);
-  width: 100vw;
   height: 3.5rem;
-  position: absolute;
-  justify-content: center;
-  align-items: center;
+  width: 100vw;
+  position: relative;
```
Comment (davidwobido) [inline src/Components/ProfilePicture/ProfilePicture.module.css]:
```suggestion
```

I don’t think you this here

File: src/Pages/Profile/Profile.module.css [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  font-size: 2.5rem;
+  padding-top: 1rem;
+}
+
+.booked {
+  color: var(--color-primary);
+  font-weight: 600;
+  font-size: 2rem;
+  margin-top: 2rem;
+  margin-bottom: 0.75rem;
```
Comment (davidwobido) [inline src/Pages/Profile/Profile.module.css]:
```suggestion
  margin: 2rem 0.75rem;
```

**Review by davidwobido — APPROVED** [review]
lgtm 💥

**Pull request merged**